cookbooks/clearwater/recipes/shared_config.rb

```diff
@@ -123,7 +123,8 @@
     ralf_session_store: ralf_session_store,
     memento_auth_store: "sprout#{site_suffix}.#{domain}",
     scscf_uri: "sip:scscf.sprout#{site_suffix}.#{domain}",
-    upstream_port: 0
+    upstream_port: 0,
+    bgcf_uri: "sip:bgcf.sprout.#{domain}"
```

Comment:
Good spot.  I'll update this (and the DNS records I assume will also need updating)